src/main/java/com/glitchcog/fontificator/bot/Message.java

```diff
@@ -414,42 +414,25 @@ private boolean isConfigChanged(ConfigMessage messageConfig, ConfigEmoji emojiCo
 
             // Bank to pull FrankerFaceZ badges from
             TypedEmojiMap ffzBadgeBank = emojiManager.getEmojiByType(EmojiType.FRANKERFACEZ_BADGE);
+            Map<String, LazyLoadEmoji> ffzBadgeMap = new LinkedHashMap<String, LazyLoadEmoji>();
```

Comment:
Does this need to be a LinkedHashMap? I don't think you depend on a consistent order for FFZ badges.